### DISCUSSION TITLE
Realistic data sets to import to standalone Mastodon for development/testing purposes?

### DISCUSSION THREAD

**Discussion by bmiller59** [mastodon/mastodon#30399] [Q&A]
Hello. I am wondering if there are any authorized data sets of realistic data that could be easily imported to a standalone Mastodon instance for development/testing purposes? I am looking for something with a typical number of users for a server with accompanying toots. Thank you.

**Comment by naught101** [comment]
I'm also wondering this - would be useful for https://github.com/mastodon/mastodon/pull/21917 (for that I need something with hashtags over a several days)

Maybe better than actual data would be a script that generates some data parametrically?
